library/LuaApi.cpp

```diff
@@ -2373,7 +2373,7 @@ static int internal_setAddress(lua_State *L)
 
     // Print via printerr, so that it is definitely logged to stderr.log.
     uintptr_t iaddr = addr - Core::getInstance().vinfo->getRebaseDelta();
-    fprintf(stderr, "Setting global '%s' to %x (%x)\n", name.c_str(), addr, iaddr);
+    fprintf(stderr, "Setting global '%s' to %zu (%zu)\n", name.c_str(), addr, iaddr);
```

Comment:
I'm not sure about `%zu` here - it does seem to be newly-supported in MSVC 2015 (yay), but wouldn't `%p` make more sense?